Source/CarthageKit/Xcode.swift

```diff
@@ -6,6 +6,17 @@ import ReactiveSwift
 import ReactiveTask
 import XCDBLD
 
+/// Gets the shared cache file path URL for dependency at pinned version
+/// ~/Library/Caches/org.carthage.CarthageKit/SharedCache/XcodeVersion/DependencyName/Version, exemple :
+/// ~/Library/Caches/org.carthage.CarthageKit/SharedCache/9.4.1_9F2000/SwiftyUserDefaults/3.0.1
+internal func sharedCacheURLByXcodeVersionFor(dependency: Dependency, pinnedVersion: PinnedVersion) -> URL {
+	let sharedCacheURL = Constants.userCachesURL.appendingPathComponent("SharedCache")
+	let sharedCachePerXcode = sharedCacheURL.appendingPathComponent(XcodeVersion.makeString(), isDirectory: true)
+	let sharedCachePerDependency = sharedCachePerXcode.appendingPathComponent(dependency.name, isDirectory: true)
+	let sharedCacheVersioned = sharedCachePerDependency.appendingPathComponent(pinnedVersion.commitish, isDirectory: true)
```

Comment:
Will this cover the case of the the commitish being a tag?

Reply:
It does, here are some tags in my local cache:
ls -1 ~/Library/Caches/org.carthage.CarthageKit/SharedCache/10.0_10A255/Alamofire/
4.7.1
4.7.3

I've built the path the same way as: https://github.com/Carthage/Carthage/blob/master/Source/CarthageKit/Project.swift#L1102-L1107

Reply:
You see an issue with this ? maybe I should add a dash and commit hash to version in the folder to manage forks (Alamofire 4.7.3 from my fork will be in the same path as the "original", but if I fork it... it might be to fix something, so I'll point to my master/fix-branch) ? In the case of a commit:
ls -1 ~/Library/Caches/org.carthage.CarthageKit/SharedCache/10.0_10A255/openium-lib.ios:
84bb9432df2efdf5bf79b108837fb4416ccc865f

Reply:
ok